src/features/category/categorySlice.js

```diff
@@ -0,0 +1,40 @@
+import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
+import axios from "axios";
+
+const initialState={
+    categories:[],
+    categoryLoader:false,
+}
+
+
+export const getCategoryList = createAsyncThunk('categoryList/getCategoryList', async (_,{rejectWithValue})=>{
+    try {
+        const res = await axios.get("/api/categories");
+        console.log(res.data.categories);
+        return res.data.categories;
+    } catch (error) {
+        console.log("from error");
```

Comment:
No need of using console debugger if you are having this rejectWithValue parameter for handling errors